lua/nvim-surround/cache.lua

```diff
@@ -6,7 +6,7 @@ local M = {}
 M.normal = {}
```

Comment:
Didn't realize until now that the type annotation on the line above is missing `delimiters: string[][]|nil`, please add that in